functions/Get-DbaQueryStoreConfig.ps1

```diff
@@ -0,0 +1,176 @@
+        $QSCDT.Columns.Add($QSCDT_MaxSize)
+        $QSCDT.Columns.Add($QSCDT_CurrentSize)
+        $QSCDT.Columns.Add($QSCDT_CaptureMode)
+        $QSCDT.Columns.Add($QSCDT_CleanupMode)
+        $QSCDT.Columns.Add($QSCDT_StaleQueryThreshold)
+	}
+
+	PROCESS
+	{
+		foreach ($servername in $SqlServer)
+		{
+            Write-Verbose "Connecting to $servername"
+			try
+			{
+				$server = Connect-SqlServer -SqlServer $servername -SqlCredential $Credential
+
+			}
+			catch
+			{
+				if ($SqlServer.count -eq 1)
+				{
+					throw $_
+				}
+				else
+				{
+					Write-Warning "Can't connect to $servername. Moving on."
+					Continue
+				}
+			}
+
+            $sqlVersion = $server.Version
+
+            $sqlVersion = $sqlVersion.ToString().Split(".")[0]
+
+            if($sqlVersion -ilt "13")
+                {
+
+                Write-Warning "The SQL Server Instance has a lower SQL Server version than SQL Server 2016. Skipping server."
+                break
+                }
+
+            
+            # We have to exclude all the system databases since they cannot have the Query Store feature enabled
+			$dbs = $server.Databases | Where-Object { $_.name -ne 'TempDb' -and $_.name -ne 'master' -and $_.name -ne 'msdb' -and $_.name -ne 'model' }
+
+			if ($databases.count -gt 0)
+			{
+				$dbs = $dbs | Where-Object { $databases -contains $_.Name }
+			}
+
+			if ($exclude.count -gt 0)
+			{
+				$dbs = $dbs | Where-Object { $exclude -notcontains $_.Name }
+			}
+
+
+			foreach ($db in $dbs)
+			{
+                $result = $null
+				Write-Verbose "Processing $($db.name) on $servername"
+
+				if ($db.IsAccessible -eq $false)
+				{
+					Write-Warning "The database $($db.name) on server $servername is not accessible. Skipping database."
+					Continue
+				}
+
+                # Start processing
+
+                $QSCDT_Row = $QSCDT.NewRow()
+
+                $QSCDT_Row.Instance = $servername
+                $QSCDT_Row.Database = $db.name
+                $QSCDT_Row.ActualState = $db.QueryStoreOptions.ActualState
+                $QSCDT_Row.FlushInterval = $db.QueryStoreOptions.DataFlushIntervalInSeconds
+                $QSCDT_Row.CollectionInterval = $db.QueryStoreOptions.StatisticsCollectionIntervalInMinutes
+                $QSCDT_Row.MaxSize = $db.QueryStoreOptions.MaxStorageSizeInMB
+                $QSCDT_Row.CurrentSize = $db.QueryStoreOptions.CurrentStorageSizeInMB
+                $QSCDT_Row.CaptureMode = $db.QueryStoreOptions.QueryCaptureMode
+                $QSCDT_Row.CleanupMode = $db.QueryStoreOptions.SizeBasedCleanupMode
+                $QSCDT_Row.StaleQueryThreshold = $db.QueryStoreOptions.StaleQueryThresholdInDays
+
+                $QSCDT.Rows.Add($QSCDT_Row)
+                       
+			}
+        }
+
+    $QSCDT | Format-Table -Property Instance, Database, ActualState, FlushInterval, CollectionInterval, MaxSize, CurrentSize, CaptureMode, CleanupMode, StaleQueryThreshold -Wrap -AutoSize 
```

Comment:
This is an interesting one, while we could add the fields we need to the datatable that is $db, I'd go with a pscustomobject

You're going to love custom objects -- very easy. No need to define column names or types. 

In this case, I'd do

[pscustomobject]@{
Instance = $servername
ActualState = $db.QueryStoreOptions.ActualState
Database = $db.name
FlushInterval = $db.QueryStoreOptions.DataFlushIntervalInSeconds
etcetc
}

etc and so on. You can order it how you need in the object itself. In addition, you do not need to assign it to any variable - just output right to the pipeline. You can see that in action here:

https://github.com/sqlcollaborative/dbatools/blob/master/functions/Get-DbaMaxMemory.ps1#L73

As for Format-Table, Shawn is correct -- the end user should be the one to use it. It turns data into something pretty but not usable. You can use Select-DefaultView to hide objects (like SMO Database objects) but they are optional.